<ui version="4.0" >
 <class>QgsAttributeTableBase</class>
 <widget class="QDialog" name="QgsAttributeTableBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Attribute Table</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>2</number>
   </property>
   <item row="0" column="0" >
    <widget class="QToolButton" name="mRemoveSelectionButton" >
     <property name="toolTip" >
      <string>Remove selection</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>../xpm/remove_selection.xpm</iconset>
     </property>
     <property name="shortcut" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QToolButton" name="mSelectedToTopButton" >
     <property name="toolTip" >
      <string>Move selected to top</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>../xpm/selected_to_top.png</iconset>
     </property>
     <property name="shortcut" >
      <string>Ctrl+T</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QToolButton" name="mInvertSelectionButton" >
     <property name="toolTip" >
      <string>Invert selection</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>../xpm/switch_selection.png</iconset>
     </property>
     <property name="shortcut" >
      <string>Ctrl+S</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QToolButton" name="mCopySelectedRowsButton" >
     <property name="toolTip" >
      <string>Copy selected rows to clipboard (Ctrl+C)</string>
     </property>
     <property name="whatsThis" >
      <string>Copies the selected rows to the clipboard</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>../xpm/copy_selection.png</iconset>
     </property>
     <property name="shortcut" >
      <string>Ctrl+C</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QToolButton" name="mZoomMapToSelectedRowsButton" >
     <property name="toolTip" >
      <string>Zoom map to the selected rows (Ctrl-F)</string>
     </property>
     <property name="whatsThis" >
      <string>Zoom map to the selected rows</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="shortcut" >
      <string>Ctrl+F</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QToolButton" name="mAddAttributeButton" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>New column</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>../xpm/new_attribute.png</iconset>
     </property>
     <property name="shortcut" >
      <string>Ctrl+N</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6" >
    <widget class="QToolButton" name="mDeleteAttributeButton" >
     <property name="toolTip" >
      <string>Delete column</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset>../xpm/delete_attribute.png</iconset>
     </property>
     <property name="shortcut" >
      <string>Ctrl+X</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7" >
    <widget class="QToolButton" name="btnEdit" >
     <property name="toolTip" >
      <string>Toggle editing mode</string>
     </property>
     <property name="whatsThis" >
      <string>Click to toggle table editing</string>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="8" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>421</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="9" >
    <widget class="QgsAttributeTable" native="1" name="tblAttributes" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="9" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Search for</string>
       </property>
       <property name="buddy" >
        <cstring>mSearchText</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mSearchText" />
     </item>
     <item>
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>in</string>
       </property>
       <property name="buddy" >
        <cstring>mSearchColumns</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mSearchColumns" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mSearchButton" >
       <property name="text" >
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mSearchShowResults" />
     </item>
     <item>
      <widget class="QPushButton" name="btnAdvancedSearch" >
       <property name="text" >
        <string>Adva&amp;nced...</string>
       </property>
       <property name="shortcut" >
        <string>Alt+N</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="9" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>QgsAttributeTable</class>
   <extends>QWidget</extends>
   <header>qgsattributetable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mRemoveSelectionButton</tabstop>
  <tabstop>mSelectedToTopButton</tabstop>
  <tabstop>mInvertSelectionButton</tabstop>
  <tabstop>mCopySelectedRowsButton</tabstop>
  <tabstop>mZoomMapToSelectedRowsButton</tabstop>
  <tabstop>mAddAttributeButton</tabstop>
  <tabstop>mDeleteAttributeButton</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>mSearchText</tabstop>
  <tabstop>mSearchColumns</tabstop>
  <tabstop>mSearchButton</tabstop>
  <tabstop>mSearchShowResults</tabstop>
  <tabstop>btnAdvancedSearch</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsAttributeTableBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>645</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel" >
     <x>682</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
